enos/ansible/group_vars/all.yml

```diff
@@ -5,6 +5,11 @@ g5k_role: unknown
 
 enable_monitoring: true
 
+# enable tc constraints when invoking tc phase
+tc_enable: true
```

Comment:
It is used to activate/deactivate  the network constraints on a node. Setting it to false will reset all rules applied on every nodes. For the moment everything is controlled from the configuration file using network_constraints.enable